packages/edit-site/src/components/sidebar-edit-mode/page-panels/page-status.js

```diff
@@ -0,0 +1,229 @@
+				newDate = new Date();
+				newDate.setDate( newDate.getDate() + 7 );
+			}
+		}
+		saveStatus( {
+			status: value,
+			date: newDate,
+		} );
+	};
+
+	return (
+		<HStack className="edit-site-summary-field">
+			<Text className="edit-site-summary-field__label">
+				{ __( 'Status' ) }
+			</Text>
+			<Dropdown
+				contentClassName="edit-site-change-status__content"
+				popoverProps={ popoverProps }
+				focusOnMount
+				ref={ setPopoverAnchor }
+				renderToggle={ ( { onToggle } ) => (
+					<Button
+						className="edit-site-summary-field__trigger"
+						variant="tertiary"
+						onClick={ onToggle }
+					>
+						<StatusLabel
+							status={ password ? 'protected' : status }
+						/>
+					</Button>
+				) }
+				renderContent={ ( { onClose } ) => (
+					<>
+						<InspectorPopoverHeader
+							title={ __( 'Status' ) }
+							onClose={ onClose }
+						/>
+						<VStack spacing={ 5 }>
+							<BaseControl
+								hideLabelFromVision
+								label={ __( 'Status' ) }
+								id={ `edit-site-change-status__status` }
+								className={ 'components-radio-control' }
+							>
+								<VStack spacing={ 3 }>
+									{ STATUS_OPTIONS.map( ( option, index ) => (
+										<div
+											key={ `edit-site-change-status__status-${ index }` }
+											className="components-radio-control__option with-hint"
+										>
+											<input
+												id={ `edit-site-change-status__status-${ index }` }
+												className="components-radio-control__input"
+												type="radio"
+												name={ `edit-site-change-status__status` }
+												value={ option.value }
+												onChange={ ( e ) =>
+													handleStatus(
+														e.target.value
+													)
+												}
+												checked={
+													option.value === status
+												}
+											/>
+											<VStack spacing={ 1 }>
+												<label
+													htmlFor={ `edit-site-change-status__status-${ index }` }
+												>
+													{ option.label }
+												</label>
+												{ option.hint && (
+													<Text variant="muted">
+														{ option.hint }
+													</Text>
+												) }
+											</VStack>
+										</div>
+									) ) }
+								</VStack>
+							</BaseControl>
+							<BaseControl
+								id={ `edit-site-change-status__password` }
+								label={ __( 'Password' ) }
+							>
+								<ToggleControl
+									label={ __(
+										'Hide this page behind a password'
+									) }
+									checked={ showPassword }
+									onChange={ handleTogglePassword }
+								/>
+								{ showPassword && (
+									<TextControl
+										onChange={ ( value ) =>
+											saveStatus( { password: value } )
+										}
+										value={ password }
+										/* eslint-disable jsx-a11y/no-autofocus */
+										autoFocus
+										/* eslint-enable jsx-a11y/no-autofocus */
+										placeholder={ __(
+											'Enter a secure password'
+										) }
+										type="password"
```

Comment:
This throws a warning in Chrome:

> [DOM] Password field is not contained in a form: (More info: (https://goo.gl/9p2vKq)

There is some great info in the link that we can take into consideration. Not sure how much is relevant here though!

Reply:
@kevin940726 would you recommend wrapping the radio inputs and password field in a form? It's tricky because we aren't actually "submitting" anything until site editor changes are saved via the save button. I'm not sure what the best practice here is.

Reply:
I'm not sure either, TBH! I guess it's still fine to wrap the whole popover in a dummy `<form>` just to please chrome?